Dockerfile

```diff
@@ -2,19 +2,31 @@
 #   Supports Debian arches: amd64, arm64, etc.
 #   Our circleci arm64 build uses this specifically.
 #   https://docs.docker.com/engine/reference/commandline/build/
-ARG arch=amd64
+#target compile arch
+ARG arch=arm64
+#host HW arch
+ARG host=amd64
 
+## Stage 0: deboostrap: setup cross-compile env 
+FROM multiarch/qemu-user-static as bootstrap
```

Comment:
docker now automatically resets the platform (so CI, user do not have to do that), allowing easy host-arch cross-compilation. 

Docker also creates whl, and tar.gz releases for PyPI and GH Releases